src/mono/wasm/Wasm.Build.Tests/Templates/WasmTemplateTests.cs

```diff
@@ -281,5 +281,24 @@ void AssertFile(string suffix)
                 Assert.True(copyOutputSymbolsToPublishDirectory == (fileName != null && File.Exists(fileName)), $"The {fileName} file {(copyOutputSymbolsToPublishDirectory ? "should" : "shouldn't")} exist in publish folder");
             }
         }
+
+        [Theory]
+        [InlineData("Microsoft.NET.Sdk", false, "<TargetIdentifier>browser-wasm</TargetIdentifier>")]
```

Comment:
```suggestion
        [InlineData("Microsoft.NET.Sdk", false, "< RuntimeIdentifier >browser-wasm</RuntimeIdentifier >")]
```

Reply:
this made it, we have an error:
`Program does not contain a static 'Main' method suitable for an entry point`
Thanks!

Reply:
btw: why do we support both: `RuntimeIdentifier` and `TargetIdentifier` and why using one has different effects than using the other?